src/kubeobjects/slice.go

```diff
@@ -42,3 +42,12 @@ func PortIsIn(ports []corev1.ContainerPort, portToCheck int32) bool {
 	}
 	return false
 }
+
+func VolumeClaimIsDefined(claims []corev1.PersistentVolumeClaim, claimNameToCheck string) bool {
+	for _, c := range claims {
```

Comment:
```suggestion
	for _, claim := range claims {
```

hard to decipher 1 letter variables unless you see the whole function
while in this case you do, but latter the diff may only show the `c.ObjectMeta.Name` in which case not so helpful 😅

Reply:
I thought the short *c* is appropriate for the ephemeral claim. And matches the rule _this variable... doesn’t have any significance outside these few lines_ and _the length of a name should correspond to the size of its scope_ stated, respectively, in *Code Complete* by Steve McConnell and *Clean Code* by Robert C. Martin.

Cloud not the operation _Expand up/down_ of _Files changed_ be helpful for a narrow diff?

Reply:
Our contribution guidline says: https://github.com/Dynatrace/dynatrace-operator/blob/master/CONTRIBUTING.md#general 
> Use descriptive names (namespace is better than ns, dynakube is better than dk, etc.)

`claim` is still short enough, `persistentVolumeClaim` would be the overkill. 😄